integration_tests/models/dbt_project_evaluator_schema_tests/graph.yml

```diff
@@ -72,3 +72,21 @@ models:
         tests:
           - unique
           - not_null
+
+  - name: stg_naming_convention_folders
+    description: "This table parses the folder name variables into a model to join into help define structure tests"
```

Comment:
Just a tiny suggestion for `stg_naming_convention_folders` and `stg_naming_convention_prefixes`.
Could we add in the docs that if someone wants to:
- change the the folder names or model prefixes, they would have to modify the `vars` in `dbt_project.yml`
- and if they have a different structure (e.g. a report or base folder and naming convention) they would need to
  1.  create new vars in `stg_naming_convention_prefixes` 
  2. create a copy of the `stg_` models locally and modify them to their needs
  3. and deactivate the equivalent `stg_` models from the package in `dbt_project.yml`
  
  Writing that makes me realize it is not a super easy flow 😄

Reply:
😅 yeah i think that is worth documenting -- do you think that is in scope for the project variables section you're writing? #102

Reply:
^capture in issue #92

Reply:
@graciegoheen are you suggesting we punt on this for this particular PR, and have a more robust docs section for using/overriding/customizing variables?